src/store/types.ts

```diff
@@ -7,3 +9,11 @@ export interface ApplicationState {
     creator: CreatorState;
     common: CommonState;
 }
+
+export interface ThunkAPI {
```

Comment:
should we have this extend `AsyncThunkConfig` from redux/tookit?

Reply:
Hm, I followed the recommended approach in the documentation here. It also looks like AsyncThunkConfig isn't exported and is a type alias rather than an interface, so it can't be extended.